### PR TITLE
[FLN-2024] Adding new organizer

### PR DESCRIPTION
**Pull request by LanusseMorais**
[FLN-2024] Adding new organizer

### CODE REVIEW

File: data/events/2024/florianopolis/main.yml [hunk 1]
```diff
@@ -82,6 +82,14 @@ team_members: # Name is the only required field for team members.
     employer: "CI&T"
     image: "rafael-martin.jpg"
     linkedin: "https://www.linkedin.com/in/rafaelmaferreira/"
+    github: "rafaelmaferreira"
+    website: "https://rafaelmaferreira.com.br/"
+
+  - name: "Gilmar Moraes"
```
Comment (phrawzty) [inline data/events/2024/florianopolis/main.yml:88]: This person isn't appearing in the [preview](https://deploy-preview-14244--devopsdays-web.netlify.app/events/2024-florianopolis/contact), but I'm not sure why. I'll get a conversation started in the website channel to debug.

**Review by phrawzty — COMMENTED** [review]
Please also see the email reply.

File: data/events/2024/florianopolis/main.yml (hunk 1 above)
Comment (mattstratton) [reply data/events/2024/florianopolis/main.yml:88]: hmm it shows in a the [preview](https://deploy-preview-14241--devopsdays-web.netlify.app/) i see in netlify (not the link @phrawzty has)

Comment (mattstratton) [reply data/events/2024/florianopolis/main.yml:88]: @phrawzty your preview link is for a different PR (the pr number is in the url/host)

Comment (phrawzty) [reply data/events/2024/florianopolis/main.yml:88]: Yep that's user error on my part. Sorry for the noise.

**Pull request merged**